spec/facades/food_facade_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe 'food facade' do
+  it 'returns an array of foods given a search string' do
+    results = FoodFacade.ingredient_search("sweet potatoes")
+
+    expect(results).to be_an(Array)
```

Comment:
Could have a few more assertions about the contents of the array to make sure you're returning those 2 items in the right order (total hits and the foods)

You could also assert that the foods array only has 10 items